src/blog/2024/04/building-an-admin-panel-in-node-red-with-dashboard-2.md

```diff
@@ -0,0 +1,108 @@
+---
+title: How to Build an Admin Dashboard with Node-RED Dashboard 2.0
+subtitle: A guide to building an Admin Panel in Node-RED with Dashboard 2.0
+description: Discover step-by-step instructions for developing an admin-only page in Node-RED Dashboard 2.0 using the FlowFuse Multiuser addon. Additionally, learn how to secure Dashboard 2.0 and explore the features of the FlowFuse multiuser addon.
+date: 2024-04-05
+authors: ["sumit-shinde"]
+image: 
```

Comment:
@sumitshinde-84 Can you create an art request issue? 

https://flowfuse.com/handbook/design/art-requests/